src/ex3/cli-todo.js

```diff
@@ -0,0 +1,117 @@
+import { Command } from "commander";
+import fetch from "node-fetch";
+import fs from "fs";
+import chalk from "chalk";
+
+const pokemonProgram = new Command();
+const path = "./todoApp.txt";
+
+async function addTodo(text) {
```

Comment:
what happened to the code from previous exercises? you could have re-use it, all the classes you've written before.
You seem to have lost some functionality such as multiple pokemons data fetch separated by a comma.